src/components/ItemList/ItemRow.tsx

```diff
@@ -12,14 +12,12 @@ export type Props = {
     onMouseEnter?: (event: React.SyntheticEvent) => void;
 };
 
-const ItemRow = ({
-    className,
-    isActive,
-    item: collaborator,
-    onClick,
-    onMouseDown,
-    onMouseEnter,
-}: Props): JSX.Element | null => {
+export type ItemRowRef = HTMLLIElement;
+
+const ItemRow = (
+    { className, isActive, item: collaborator, onClick, onMouseDown, onMouseEnter }: Props,
```

Comment:
Can we destructure props within the function to avoid this gnarly function signature?